src/qml/ChatPage.qml

```diff
@@ -59,18 +59,23 @@ Kirigami.Page {
 				Rectangle {
 					id: avatar
 
-					width: height
-					height: parent.height
+					width: 40
+					height: 40
+					radius: width*0.5
 					color: "grey"
 					visible: !sentByMe
 				}
 
 				Rectangle {
-					width: 80
-					height: 40
+					radius: 2
```

Comment:
Ok, I wasn't sure about what you and @MRZA-MRZA would say about this design, so I decided to do only a small change here.